taskcluster/scripts/toolchain/external-gradle-dependencies/after.sh

```diff
@@ -32,4 +33,18 @@ function _package_artifacts_downloaded_by_nexus() {
 }
 
 
+function _ensure_artifacts_are_sane() {
+    : > "$BAD_DOWNLOADS_FILE"
+    # Let's find empty files or unfinished downloads
+    find "$WORKSPACE/$EXTERNAL_DEPS" -size 0 -o -name '*.part' >> "$BAD_DOWNLOADS_FILE"
+
+    if [[ -s "$BAD_DOWNLOADS_FILE" ]]; then
+        echo "ERROR: Some artifacts were not correctly downloaded! Please look at:"
+        cat "$BAD_DOWNLOADS_FILE"
+        exit 17
```

Comment:
Maybe also move the 17 to an appropriately-named variable or add a comment to explain to future-us it's to cause an automatic rerun?

Reply:
Good call. Done.